Content.Client/Materials/UI/MaterialDisplay.xaml.cs

```diff
@@ -71,7 +71,7 @@ public void PopulateButtons(int maxEjectableSheets)
         if (!_canEject)
             return;
 
-        int[] sheetsToEjectArray = { 1, 5, 10 };
+        int[] sheetsToEjectArray = { 1, 5, 10, 30 }; // Frontier
```

Comment:
```suggestion
        int[] sheetsToEjectArray = { 1, 5, 10, 30 }; // Frontier: add 30
```
Comment what changed.